convert2rhel/checks.py

```diff
@@ -370,14 +370,27 @@ def ensure_compatibility_of_kmods():
     if not unsupported_kmods:
         logger.debug("All loaded kernel modules are available in RHEL.")
     else:
-        logger.critical(
-            "The following loaded kernel modules are not available in RHEL:\n{0}\n"
-            "First, make sure you have updated the kernel to the latest available version and rebooted the system.\n"
-            "If this message appears again after doing the above, prevent the modules from loading by following {1}"
-            " and run convert2rhel again to continue with the conversion.".format(
-                "\n".join(unsupported_kmods), LINK_PREVENT_KMODS_FROM_LOADING
+
+        unsupported_kmods = get_unsupported_kmods(host_kmods, rhel_supported_kmods)
```

Comment:
Oh, now I realize that we have this line above this if-else statement... I think you can remove it safely now, as we already have the unsupported kmods available. 

I believe it was me that asked you to place it here, so my bad. 
```suggestion

```

Reply:
gotcha I can definitely make that change and no worries at all!